mcs/class/corlib/System/TimeZoneInfo.cs

```diff
@@ -1337,6 +1337,18 @@ static int ReadBigEndianInt32 (byte [] buffer, int start)
 			return SwapInt32 (i);
 		}
 
+		static long ReadBigEndianInt64(byte[] buffer, int start)
+		{
+			byte[] longBytes = new byte[8];
+			for (int i = 0; i < 8; i++)
```

Comment:
It's a "style" remark, but I'd enclose the loop body (as well as single-line bodies of `if` statements below) in braces.  This avoids indentation-based mistakes